plugin/editor/blocks/text-block/index.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * External dependencies
+ */
+import { createElement } from '@wordpress/elements';
+import { registerBlock, Editable } from '@wordpress/blocks';
+
+registerBlock( 'wp', 'Text', {
+	edit( state, onChange ) {
+		return createElement( Editable, {
+			value: state.value,
+			onChange: ( value ) => onChange( { value } )
+		} );
+	},
+	save( state ) {
+		return createElement( 'p', null, state.value );
```

Comment:
Why do we need multiple return types? String could work for all use-cases, right?
Why do we need it to be an element (which is HTML + events instead of just HTML)?